tests/main/interfaces-audio-playback-record/task.yaml

```diff
@@ -35,19 +35,21 @@ prepare: |
     session-tool -u test --prepare
     echo "session-tool -u test --restore" >>defer.sh
 
+    # Ensure that the socket is active but do not check the service.  For some
+    # reason, pulseaudio quits as soon as it is not needed and this means that
+    # it works when we use paplay but shuts down relatively quickly after
+    # playback. We use to run "pulseaudio --check" but this only worked when
+    # pulse was still running, just before quitting, after the session was
+    # prepared a split second before.
+    session-tool -u test systemctl --user is-active pulseaudio.socket | MATCH active
```

Comment:
Is there any guarantee that this unit will have finished starting at this point?  I definitely observed races with the user session starting up in some tests before porting them to session-tool, and it's not obvious that session-tool would fix that.

If you want to wait until pulseaudio.socket is active, how about the following?

    session-tool -u test systemctl --user start pulseaudio.socket

Reply:
Interesting, perhaps the socket startup is also racy but because systemd --user starts up during PAM setup and sockets are relatively inexpensive, we never got to observe one.

Perhaps instead we just want to have session-tool --prepare wait for default.target?

Reply:
We're now starting `default.target` in `session-tool --prepare`